azure/coverage.yml

```diff
@@ -1,13 +1,18 @@
 parameters:
   envs: {}
+  nightly: false
 
 jobs:
  - job: tarpaulin
    displayName: tarpaulin
    pool:
      vmImage: ubuntu-16.04
+   continueOnError: ${{ parameters.nightly }}
```

Comment:
Is there a reason we're coupling these concepts?

Reply:
Yes -- nightly is notoriously finicky, especially when not pinned to a particular nightly date. Not allowing failures (yellow CI) on nightly is just begging for spurious red CI, which I think we want to avoid at almost all cost.

Reply:
Isn't that a case for supporting pinning nightly?  The times I've relied on unstable features (just clippy and rustfmt), I've pinned nightly.

This reminds me of my original concern over providing a one-stop pipeline. Satisfying everyone is hard; there are a lot of different corner cases.

Reply:
Ah, no, the nightly here is dictated by the tarpaulin Docker image, so we can't actually pin our own :) Otherwise, yes, normally that's what you'd do.

Reply:
Should we record this assumption?  comment?  commit message?

Reply:
Documented in the documentation I added in https://github.com/crate-ci/azure-pipelines/pull/42/commits/f177a29c5eca625e5533b07d75aace44fae14bf7. Do you think that is sufficient?

Reply:
Or in the public documentation, sure